client/src/api/plugin/Queries.tsx

```diff
@@ -1,14 +1,19 @@
 import { useQuery } from '@tanstack/react-query';
 import axios from 'axios';
-import { PluginDefinition } from '../Models';
+import { PluginDefinition, PluginParameters } from '../Models';
 
 export function useGetPlugins() {
   return useQuery<PluginDefinition[]>(
     ['plugins'],
     async () => {
       const response = await axios.get<PluginDefinition[]>('/api/plugins').catch((err) => {
-        console.log('useGetPlugins error', err);
-        return { data: [] };
+        console.error('Failing getting plugins from api', err);
+        if (import.meta.env.IQENGINE_PLUGINS) {
+          const plugins = JSON.parse(import.meta.env.IQENGINE_PLUGINS);
```

Comment:
You probably want to check whether this parses correctly and not use this value if it doesn't, or give some indication it doesn't parse correctly